### PR TITLE
New mailgun pack

### PR DESCRIPTION
**Pull request by Kami**
It initially contains an action for sending an email via Mailgun HTTP API.

### CODE REVIEW

File: packs/mailgun/actions/send_email.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from st2actions.runners.pythonrunner import Action
+
+__all__ = [
+    'SendEmailAction'
+]
+
+SEND_EMAIL_API_URL = 'https://api.mailgun.net/v2/%(domain)s/messages'
+
+
+class SendEmailAction(Action):
```
Comment (jfryman) [inline packs/mailgun/actions/send_email.py]: Questions for myself here... Let's say I have an integration with another Mail sender (Postmark, Amazon SES, whatever), and the pack action also uses the same class name. What happens? Should we care about namespacing here?

Comment (Kami) [inline packs/mailgun/actions/send_email.py]: Class names are local to the pack and the file so explicit namespacing is not necessary - as long as the class is in a different file and / or pack, there should be no problems.

Same goes for sensors, although sensors are a bit different - they explicitly specify class name in the metadata file, where in the python-runner action case, we just pull out the first class from the file (I'd actually like to make this more explicit and consistent with the sensor, eventually...).

Comment (jfryman) [inline packs/mailgun/actions/send_email.py]: :cool:. Thanks.

**Comment by jfryman** [comment]
:+1:

File: packs/mailgun/pack.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name : mailgun
+description : st2 content pack containing mailgun integrations
+keywords:
+  - email
+  - mail
+  - mailgun
+version : 0.1
+author : st2-dev
+email : info@stackstorm.com
```
Comment (lakshmi-kannan) [inline packs/mailgun/pack.yaml]: I usually use support@stackstorm.com

File: packs/mailgun/actions/send_email.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            'subject': subject,
+            'text': text
+        }
+
+        if html:
+            data['html'] = html
+
+        api_url = SEND_EMAIL_API_URL % {'domain': domain}
+        response = requests.post(api_url, auth=('api', api_key), data=data)
+        return response.status_code == httplib.OK
```
Comment (lakshmi-kannan) [inline packs/mailgun/actions/send_email.py]: I am not sure returning False fails the action. I think you might need to raise an exception. Can you double check this part?

Comment (Kami) [inline packs/mailgun/actions/send_email.py]: Yeah, I don't think so either.

And a good point overall - we do follow this `return True / False` pattern in other cases as well, but I agree with you and think we should throw if an action fails so this is reported correctly up the chain.

Comment (Kami) [inline packs/mailgun/actions/send_email.py]: Pushed a fix in d6c22437a47a6f34213dd386a9e40110c64a8a12.

**Pull request merged**